slashCommands/aoc.ts

```diff
@@ -3,6 +3,9 @@ import {CommandInteraction} from "discord.js";
 import {embedLeaderboard} from "../lib/aocleaderboardparser";
 import {get} from "../lib/configmanager";
 
+/**
+ * Slash command definition for /aoc, a command that sends the current leaderboard
```

Comment:
All Slash-Command-Docs have the same structure except timetable (Wirrten like it is here). Should we make that uniform.